cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -256,6 +260,7 @@ public Module getDistributedModules() {
                            new ConfigStoreModule(),
                            new EntityVerifierModule(),
                            new ProvisionerModule(),
+                           new TetheringModule(),
```

Comment:
Is this still needed? Seems like the handler classes are binding multiple times? Also, why it is only for the distributed module? Can I use two standalone to test tethering?

Reply:
I think it's not needed. Removed TetheringModule